dataclients/kubernetes/testdata/ingressV1/ingress-data/ing-with-duplicate-host-different-path.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: foo
+  name: qux
+spec:
+  defaultBackend:
+    service:
+      name: bar
+      port:
+        number: 1234
+  rules:
+  - host: www.example.org
+    http:
+      paths:
+      - path: "/"
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: bar
+            port:
+              name: baz
+  - host: www.example.org
+    http:
+      paths:
+      - path: "/test"
+        pathType: ImplementationSpecific
+        backend:
+          service:
+            name: bar
+            port:
+              name: baz
```

Comment:
This actually looks like a bug - they should not have the same id

Reply:
The don't actually.

Reply:
It's to ensure duplicate host != duplicate routeID, will add another test case for different services.